packages/cozy-sharing/src/components/Recipient.jsx

```diff
@@ -182,12 +182,18 @@ const PermissionsWithBreakpoints = withBreakpoints()(
 
 const Status = ({ status, isMe, instance }) => {
   const { t } = useI18n()
-  const shouldShowStatus = ['pending', 'seen'].includes(status) && !isMe
-  const text = shouldShowStatus ? t(`Share.status.${status}`) : instance
 
-  let icon = 'to-the-cloud'
-  if (shouldShowStatus && status === 'pending') icon = 'paperplane'
-  else if (shouldShowStatus && status === 'seen') icon = 'eye'
+  let text, icon
+  if (isMe) {
+    text = instance
+    icon = 'to-the-cloud'
+  } else {
+    text = ['pending', 'seen'].includes(status)
```

Comment:
const supportedStatus = ['pending', 'seen']... 

Will be easier to understand and maintain if needed